docs/guides/references/changelog.mdx

```diff
@@ -11,7 +46,7 @@ _Released 05/23/2023_
 - Adds Git-related messages for the
```

Comment:
```suggestion
- Added Git-related messages for the
```

Reply:
@emilyrohrbough Is there a standard on verb tense that is supposed to be used?  I looked at the changelog guide and did not see that mentioned. For this particular line, it was originally `Add`, then got changed to `Adds`, and now `Added`.  Looking at older entires, they seem go back and forth with verb tense.

Reply:
I think it's meant to be past tense. But good thing to prob call call out